.Rprofile

```diff
@@ -13,8 +13,71 @@ if (isTRUE(rownames(installed.packages(priority = "NA")) == "renv")) {
   message("Finished installing R package dependencies.")
 }
 
-# source global .Rprofile (very important to load user specific settings)
-# DO NOT EDIT THIS LINE!
-if (file.exists("~/.Rprofile")) {
-  source("~/.Rprofile")
+base::local({
```

Comment:
```suggestion
local({
```
Prefixing with `base` looks weird, I just wrote it with `base::` to emphasize that it is just available anyway :slightly_smiling_face: